nz.mega.MEGAsync.yml

```diff
@@ -98,15 +98,16 @@ modules:
           - /bin
 
       - name: freeimage
+        subdir: FreeImage/trunk
         no-autogen: true
         make-install-args:
           - DESTDIR=/app
           - INCDIR=/app/include
           - INSTALLDIR=/app/lib
         sources:
-          - type: archive
-            url: https://sourceforge.net/code-snapshots/svn/f/fr/freeimage/svn/freeimage-svn-r1859-FreeImage-trunk.zip
-            sha256: 29f01d45ac81246edfcb3bc0c66f996828f7b18624787e2f5e0558effe79d5f3
+          - type: svn
+            url: https://svn.code.sf.net/p/freeimage/svn/
+            revision: r1859
```

Comment:
If this is to build on `aarch64`, there is an easier fix using the release.

All you need is to add
```yaml
  build-options:
      cflags: -fPIC
      cxxflags: -fPIC
```

(maybe conditional for aarch64)